examples/cerebro/clearnet/clearnode_client.go

```diff
@@ -1,472 +1,247 @@
 package clearnet
 
 import (
-	"encoding/json"
+	"context"
 	"fmt"
-	"net"
-	"sync"
-	"time"
 
-	"github.com/ethereum/go-ethereum/common"
-	"github.com/gorilla/websocket"
 	"github.com/shopspring/decimal"
 
-	"github.com/erc7824/nitrolite/examples/cerebro/unisig"
-)
-
-const (
-	pingRequestID = 100             // Request ID for ping messages
-	pingInterval  = 5 * time.Second // Interval for ping messages
+	"github.com/erc7824/nitrolite/clearnode/pkg/rpc"
+	"github.com/erc7824/nitrolite/clearnode/pkg/sign"
 )
 
 type ClearnodeClient struct {
-	conn   *websocket.Conn
-	signer unisig.Signer // User's Signer
-
-	printEvents   bool
-	responseSinks map[uint64]chan *RPCResponse // Map of request IDs to response channels
-	mu            sync.RWMutex                 // Mutex to protect access to responseSinks
-	exitCh        chan struct{}                // Channel to signal client exit
-}
+	rpcDialer rpc.Dialer
+	rpcClient *rpc.Client
+	signer    sign.Signer // User's Session Key
```

Comment:
Let's name this field to `sessionKey` if it is already commented this way.

Reply:
[8cd2501](https://github.com/erc7824/nitrolite/pull/410/commits/8cd2501e6d3c9275c6fc629eeea343133014e9ed)